std/experimental/ndslice/slice.d

```diff
@@ -1249,7 +1256,7 @@ struct Slice(size_t _N, _Range)
         if and only if all strides are positive.
     +/
     static if (is(PureRange == Range))
-    auto ptr() @property
+    auto ptr() @property @system
```

Comment:
That is extremely scary. What you're basically saying is that the struct is free to do anything at all and we blindly accept it as safe.

While it may be OK right now, what happens when people change the code later? They may accidentally introduce bugs that break safety, and the compiler won't even notice. That's bad.  If this is the case we may need to look into redesigning things so that we don't have to blindly trust an entire struct to do the right thing.

Basically `@trusted` should be applied to as little code as possible -- mainly the bits that the compiler cannot verify is safe even though we know better.  Everything else should be something the compiler can automatically check is `@safe`.